bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/GarbageCollectorThread.java

```diff
@@ -462,16 +463,23 @@ private void doGcEntryLogs() throws EntryLogMetadataMapException {
         this.numActiveEntryLogs = entryLogMetaMap.size();
     }
 
-    private void removeIfLedgerNotExists(EntryLogMetadata meta) throws EntryLogMetadataMapException {
+    private boolean removeIfLedgerNotExists(EntryLogMetadata meta) throws EntryLogMetadataMapException {
+        AtomicBoolean modified = new AtomicBoolean(false);
```

Comment:
you can use light weight `MutableBoolean` instead `AtomicBoolean`

Reply:
done